druid/src/lens.rs

```diff
@@ -293,6 +293,75 @@ where
     }
 }
 
+/// A wrapper for widgets that don't use data, i.e. `Widget<()>`.
```

Comment:
I'm not sure that `lens` is quite the right place for this? I get that it's _kind_ of a lens, like a specialization of one particular lens, but still not 100% clear. 🤔